src/App.js

```diff
@@ -1,16 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [entriesData, setEntriesData] = useState(chatMessages);
+
+  const calcLikes = (entriesData) => {
+    let likeCount = 0;
+
+    for (const entry of entriesData) {
+      if (entry.liked) {
+        likeCount++;
+      }
+    }
+    return likeCount;
+  };
```

Comment:
👍  I like that you encapsulate this logic into a method and have the method execute when the JSX renders instead of javascript code sitting out in the open in App.